apis/poloniex/public/chart data fetcher/545da653-7f37-4f66-aa7f-8d0354d10bec.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "apis/poloniex/public/Chart data fetcher",
+  "description": "Returns candlestick chart data. Required GET parameters are \"currencyPair\", \"period\" (candlestick period in seconds; valid values are 300, 900, 1800, 7200, 14400, and 86400), \"start\", and \"end\". \"Start\" and \"end\" are given in UNIX timestamp format and used to specify the date range for the data returned.\n\nhttps://docs.poloniex.com/#returntradehistory-public",
```

Comment:
The url takes you wrong api doc. Right one: https://docs.poloniex.com/#returnchartdata